CppSamples/ChilliConnectCppDemo/Main.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+#include <time.h> 
+#include "ChilliConnect.h"
+
+using namespace std;
+
+int main()
+{
+	//Replace these with the relevant values from your own dashboard
+	const string gameToken = "<GAME-TOKEN>";
+	const string leaderboardKey = "MY_LEADERBOARD";
+
+	//Create a new instance of the ChilliConnect client
+	ChilliConnect * chilliConnect = new ChilliConnect(gameToken);
```

Comment:
You create this and then don't delete it. You prob don't need to create this on the heap (i.e. with new) you can create it as ChilliConnect chilliConnect(gameToken). If you do need to create it on the heap you can delete it at the end or create it as a unique_ptr which will auto delete when it goes out of scope